mobilecoind/clients/python/cli/check_balance.py

```diff
@@ -52,7 +52,7 @@
     print("    {:<18}{}".format("Master Key:", args.key))
     print("    {:<18}{}".format("Subaddress Index:", args.subaddress))
     print("    {:<18}{}".format("Address Code:", public_address.b58_code))
-    print("    {:<18}{}".format("Address URL:", public_address.mob_url))
+    print("    {:<18}{}".format("Address URL:", "mob58://"+ public_address.b58_code))
```

Comment:
`mob58://`? Why not just have it print out the b58 code without the url prefix?

Reply:
Actually, I see that it prints it out on the line before, so why not just remove this line entirely?

Reply:
we've decided that our convention for deeplinks will be to pass a b58 code with the URI "mob58://"